cmd/nerdctl/ps_linux_test.go

```diff
@@ -36,11 +43,22 @@ func prepareTest(t *testing.T) (*testutil.Base, string) {
 		base.Cmd("rm", "-f", testContainerName).AssertOK()
 	})
 
+	testLabels := make(map[string]string)
+	keys := []string{
+		testutil.Identifier(t),
+		testutil.Identifier(t),
+	}
```

Comment:
Why repeat same key twice?

Reply:
I think multiple labels can exist.
For example, `--label key1=value1,key2=value2`
So, the test makes multiple labels.

Reply:
Could you clarify that intent in the comment lines of the code?

Reply:
I commented.
Could you please check again?

I'm really sorry about it. That's all my fault.